docs/tutorials/integrations/nginx.md

```diff
@@ -0,0 +1,374 @@
+
+## Using NGINX To Proxy Traffic With TLS To Conjur
+
+[NGINX][NGINX] is a web proxy that's Free Software and easy to configure. This
+tutorial will show you how to use Docker to install Conjur and NGINX and
+configure them to provide Conjur with TLS.
+
+[NGINX]: https://www.nginx.com
+
+### Prerequisites
+
+This tutorial requires Docker and a terminal application. Prepare by following
+the prerequisite instructions found on [Install Conjur][prerequisites].
+
+[prerequisites]: /get-started/install-conjur.html#prerequisites
+
+Additionally, you will need the tutorial files from the Conjur source code
+repository. Here's how you get them:
+
+1. Install Git: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+1. Clone the Conjur repository
+   
+   In your terminal application, run:
+   ```sh-session
+   $ git clone https://github.com/cyberark/conjur.git
+   ```
+   This will create a folder called `conjur` in the current directory.
+
+### The Good Part
+
+To start out our experience on a high note, let's get the full Conjur+TLS stack
+up and running so we can inspect it.
+
+The tutorial script will install Conjur and NGINX, configure them to work
+together, and connect a Conjur client to the running server. This is a full
+end-to-end working installation to allow you to see how the pieces fit.
+
+```sh-session
+$ # start the Conjur+NGINX tutorial servers
+$ cd conjur/docs/tutorials/integrations/nginx
+$ ./start.sh
+```
+
+Take a look at the logs for the Conjur or NGINX servers:
+
+```sh-session
+$ # in the 'nginx' directory we navigated to previously:
+$ docker-compose logs conjur
+$ docker-compose logs proxy
+```
+
+Run some commands in the Conjur client:
+
+```sh-session
+$ # in the 'nginx' directory we navigated to previously:
+$ docker-compose exec client bash
+# conjur authn whoami
+# conjur list
+```
+
+#### What's happening here?
+
+When you read the Conjur access logs, you'll note that the Conjur server is
+listening on port 80 (*insecure http*) inside its container. However, that port
+is not exposed except on the local Docker network, so requests from the Internet
+and LAN are unable to reach it.
+
+Meanwhile, the NGINX container is exposing its port 443 (*https*) to the outside
+network and proxying the traffic through to Conjur.
+
+## Breaking Down the Tutorial
+
+These files show how the proxy setup works. Note that, while this tutorial uses
+Docker containers and NGINX, there's nothing magic about those technologies. You
+can replicate the same strategy using a different endpoint such as
+[HAProxy][haproxy-tls] and services that run in virtual machines or on bare
+metal. To create a tutorial that can run as conveniently on a laptop as in the
+cloud, we provide this setup.
+
+[haproxy-tls]: https://www.haproxy.com/documentation/aloha/7-0/haproxy/tls/
+
+### docker-compose.yml
+
+This file declares services to be used in the tutorial. Let's break down each declaration:
```

Comment:
This may be beyond the scope, but it would be cool if we included a simple architectural diagram (get those visual learners to quickly see what's going on here)

Reply:
I would like to do this, too. That's on the list for the next time we revise this.